content/en/docs/reference/android.md

```diff
@@ -44,8 +44,12 @@ When an authenticator is not persistently linked, a QR code must be scanned on e
 
 - The [Device Public Key (DPK)](../terms#device-public-key-dpk) extension is supported in beta, but is currently gated behind a flag in Chrome. Developers can enable `chrome://flags/#enable-experimental-web-platform-features` to experiment with DPK on Android, or on desktop Chrome when using [Cross-Device Authentication](../terms#cross-device-authentication-cda) with an Android device.
 
-## Resources
+## Official Resources
```

Comment:
Please revert this, all other documents use this header level and "Resources" and I'd want a renaming like this to update all pages at the same time.